app/policies/application_policy.rb

```diff
@@ -63,6 +63,10 @@ def minimal_admin?
     user.has_role?(:super_admin) || user.has_role?(:admin)
   end
 
+  def intern_admin?
+    user.has_role?(:intern) || user.has_role?(:super_admin) || user.has_role?(:admin)
```

Comment:
@benhalpern made some changes but I don't think it's what you meant..
I also removed one of the Role validations because I couldn't figure out how to add "Article" as a resource type. Right now, if you query for `Rolify.resource_types` it returns an empty array.

Reply:
What Ben is suggestion is using `somebody.add_role(:admin, Article)` can achieve the same goal without creating a brand new role. 

To make this happen, you need to [resourcify](https://github.com/RolifyCommunity/rolify#32-configure-your-resource-models) the article model first. Then you can check user's permission with `somebody.has_role?(:admin, Article)`.